### PR TITLE
Upgrade test for employee ID and find

### PR DESCRIPTION
**Pull request by kevinchangjk**
Upgraded testing for:

- Employee ID
  - Added testing for pretty much everything that ID does
  - If any breaking changes are made to ID, this is the first sign of failure
  - Also tried to fix the "sometimes pass sometimes fail" test cases (will see if it actually works)
- Find command
  - Added more test cases for parser as well

### CODE REVIEW

**Review by prit3010 — COMMENTED** [review]
Oops

File: src/test/java/coydir/logic/parser/FindCommandParserTest.java [hunk 1]
```diff
@@ -15,7 +16,18 @@ public class FindCommandParserTest {
...
     @Test
     public void parse_emptyArg_throwsParseException() {
-        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_invalidArgs_throwsParseException() {
+        assertParseFailure(parser, "n/Prittam",
```
Comment (Kok-je) [inline src/test/java/coydir/logic/parser/FindCommandParserTest.java:25]: Nice. Will be good to write some comments to describe the scenario you are testing

**Review by Kok-je — COMMENTED** [review]
LGTM

File: src/main/java/coydir/logic/parser/FindCommandParser.java [hunk 2]
```diff
@@ -43,9 +42,6 @@ public FindCommand parse(String args) throws ParseException {
         }
         if (argMultimap.getValue(PREFIX_POSITION).isPresent()) {
             keywordPosition = ParserUtil.parseKeyword(argMultimap.getValue(PREFIX_POSITION).get());
-            if (!Position.isValidPosition(keywordPosition)) {
```
[the remark this answers is not in the document]
Comment (kevinchangjk) [reply src/main/java/coydir/logic/parser/FindCommandParser.java:46]: Yes.

**Review by Kok-je — COMMENTED** [review]
LGTM

**Pull request merged**